Softeq.XToolkit.WhiteLabel.sln

```diff
@@ -23,6 +23,12 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Softeq.XToolkit.Bindings.iO
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Softeq.XToolkit.Common.iOS", "Softeq.XToolkit.Common.iOS\Softeq.XToolkit.Common.iOS.csproj", "{6BCB2009-2E46-458C-BCAA-AFC27A631924}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Softeq.XToolkit.PushNotifications", "Softeq.XToolkit.PushNotifications\Softeq.XToolkit.PushNotifications.csproj", "{11BC7FFF-88E1-4043-A404-7B4907757FB2}"

```

Comment:
Please add projects references to Playground solution

Reply:
done